app/lib/common/widget/FaqListItem.dart

```diff
@@ -0,0 +1,319 @@
+                                              msg: 'Heart icon tapped',
+                                              toastLength: Toast.LENGTH_SHORT,
+                                            )
+                                          },
+                                          child: Image.asset(
+                                            'assets/images/heart_like.png',
+                                          ),
+                                        ),
+                                        Padding(
+                                          padding: const EdgeInsets.only(
+                                            left: 8.0,
+                                          ),
+                                          child: Text(
+                                            '189',
+                                            style: GoogleFonts.montserrat(
+                                              color: Colors.white,
+                                              fontSize: 16,
+                                            ),
+                                          ),
+                                        ),
+                                        Padding(
+                                          padding: EdgeInsets.only(left: 8.0),
+                                          child: GestureDetector(
+                                            onTap: () => {
+                                              Fluttertoast.showToast(
+                                                msg: 'Comment icon tapped',
+                                                toastLength: Toast.LENGTH_SHORT,
+                                              )
+                                            },
+                                            child: Image.asset(
+                                              'assets/images/comment.png',
+                                            ),
+                                          ),
+                                        ),
+                                        Padding(
+                                          padding: EdgeInsets.only(left: 8.0),
+                                          child: Text(
+                                            '32',
+                                            style: GoogleFonts.montserrat(
+                                              color: Colors.white,
+                                              fontSize: 16,
+                                              fontWeight: FontWeight.w600,
+                                            ),
+                                          ),
+                                        ),
+                                      ],
+                                    ),
+                                  ),
+                                )
+                              ],
+                            ),
+                          ),
+                          Padding(
```

Comment:
this, too, is a list of comments and as such should a) be mocked from the native and not in the ui and b) be a separate component that can be reused in other places (and c) use the material theme system not static colors).